spec/javascripts/packages/document-capture/components/document-capture-spec.jsx

```diff
@@ -158,6 +158,7 @@ describe('document-capture/components/document-capture', () => {
 
     await new Promise((resolve) => {
       onSubmit.callsFake(resolve);
+      // eslint-disable-next-line no-restricted-syntax
```

Comment:
These are fine to not `await`, since we're indirectly awaiting the result through the mocked `onSubmit` handling which is assumed to eventually be called as a result of the click.